applications/poisson/overset_grids/input.json

```diff
@@ -0,0 +1,18 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "2",
+        "IsTest": "false"
+    },
+    "SpatialResolution": {
+        "DegreeMin": "3",
+        "DegreeMax": "3",
+        "RefineSpaceMin": "2",
+        "RefineSpaceMax": "2"
```

Comment:
Why not specify these independently for the domains?

Reply:
this is something I could change already for the present PR

Reply:
done here https://github.com/exadg/exadg/pull/186/commits/fa5eb24dfa3c6633fbf487f7b7d8e8954bd32165